src/java/org/apache/cassandra/triggers/CustomClassLoader.java

```diff
@@ -83,7 +82,7 @@ public void addClassPath(File dir)
             logger.info("Loading new jar {}", inputJar.absolutePath());
             try
             {
-                copy(inputJar.toPath(), out.toPath());
+                copy(inputJar.toPath(), out.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
because `FileUtils.createTempFile("cassandra-", ".jar", lib);` creates an empty file, copying here will fail because it will not copy it when the destination exists. We need to replace it.